src/pages/workspaces/workspace/jobHistory/CallCacheWizard.js

```diff
@@ -0,0 +1,270 @@
+            div({ style: { paddingRight: '0.5rem' } }, ['Attempt:']),
+            (!!otherCallFqn && otherIndex !== undefined) ?
+              div({ style: { paddingRight: '0.5rem', flex: '2 1 auto' } }, [otherCallAttemptDecision(otherWorkflowMetadata, otherCallFqn, otherIndex)]) :
+              'Select a call FQN and an index first'
+          ]),
+          !!(otherCallFqn && otherIndex !== undefined && otherAttemptSucceeded) && h(ButtonPrimary, { style: { float: 'right' }, onClick: () => { fetchDiff(otherWorkflowId, otherCallFqn, otherIndex); setOtherCallSelected(true) } }, ['Compare Diff >'])
+        ]) :
+        'Loading other workflow calls...'),
+      divider,
+      div({ style: { paddingTop: '0.5rem', fontSize: 16, fontWeight: 500, color: 'lightgray', ...Style.noWrapEllipsis } }, ['Result: View cache diff'])
+    ])
+  }
+
+  const compareDiffs = () => {
+    return h(Fragment, [
+      div({ style: { paddingBottom: '0.5rem', display: 'flex', alignItems: 'center', fontSize: 16, fontWeight: 500 } }, [
+        div(['Selected workflow B: ']),
+        div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'center', flex: '1 1 100px' } }, [
+          div({ style: { marginLeft: '0.5rem', ...Style.noWrapEllipsis, ...Style.codeFont } }, otherWorkflowId)
+        ]),
+        h(ButtonSecondary, { style: { textTransform: 'none', paddingLeft: '1rem', height: '20px', justifyContent: 'right' }, onClick: () => resetWorkflowSelection() }, ['Reset'])
+      ]),
+      div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'center', fontSize: 16, fontWeight: 500 } }, [
+        div(['Selected call B: ']),
+        div({ style: { display: 'flex', flexDirection: 'row', alignItems: 'center', flex: '1 1 100px' } }, [
+          div({ style: { marginLeft: '0.5rem', ...Style.noWrapEllipsis, ...Style.codeFont } }, otherCallFqn),
+          breadcrumbHistoryCaret,
+          div({ style: { marginLeft: '0.5rem', ...Style.noWrapEllipsis, ...Style.codeFont } }, `index ${otherIndex}`)
+        ]),
+        h(ButtonSecondary, { style: { textTransform: 'none', paddingLeft: '1rem', height: '20px', justifyContent: 'right' }, onClick: () => resetCallSelection() }, ['Reset'])
+      ]),
+      divider,
+      div({ style: { display: 'flex', alignItems: 'center', fontSize: 16, fontWeight: 500 } }, ['Result: View cache diff']),
+      diffError ?
+        h(ErrorView, { error: diffError }) :
+        (diff ?
+          [div({ style: { marginTop: '0.5rem', marginBottom: '0.5rem' } }, ['Note: the diff is expressed in terms of hashes of values rather than raw values because it is hashes that determine cache hits.']),
+            h(ReactJson, {
+              style: { whiteSpace: 'pre-wrap', border: 'ridge', padding: '0.5rem' },
+              name: false,
+              shouldCollapse: ({ name }) => { return name === 'callA' || name === 'callB' },
```

Comment:
nit: Braceless expression would be preferred here, e.g.
```suggestion
              shouldCollapse: ({ name }) => name === 'callA' || name === 'callB',
```